validation/sequence_provider.rst

```diff
@@ -139,6 +139,14 @@ that group are valid, the second group, ``Strict``, will be validated.
     infinite recursion (as the ``Default`` group references the group
     sequence, which will contain the ``Default`` group which references the
     same group sequence, ...).
+    
+.. caution::
+    
+    Calling ``validate()`` with a group in the sequence (``Strict`` in previous example)
+    will cause a validation **only** with that group and not with all the groups in the 
+    sequence.
+    This is because sequence is now referred to ``Default`` group validation even if the
```

Comment:
I think you could end this sentence by adding a dot after "validation" here.
I'm not the sure of the added value of the rest which sounds more confusing after reading the caution.

Reply:
I've updated the PR. Thanks for the feedback :)